api/v1alpha1/remoteWrite_types.go

```diff
@@ -17,6 +17,16 @@ type RemoteWrite struct {
 type RemoteWriteSpec struct {
 	RemoteWrite     pov1.RemoteWriteSpec `json:"remoteWrite"`
 	ClusterSelector metav1.LabelSelector `json:"clusterSelector"`
+
+	// Secrets that will be created in the relevant Prometheus namespace
```

Comment:
I would change the description to something like

```suggestion
	// Secrets data to be created along with the configured Prometheus resource.
	// This provides the data for any v1.SecretKeySelector used in the subsequent RemoteWrite field.
	// Provided name and keys should match values in v1.SecretKeySelector fields.
```

I am also thinking we should provide an example somewhere to illustrate how this works.

Reply:
https://github.com/giantswarm/prometheus-meta-operator/pull/928/files#diff-a8f899db168f9bb204ce8254693b9911f9e06e693f5647e447a4df2b828b54f9R28

Reply:
Nice example :+1: